packages/vuetify/src/components/VDialog/VDialog.js

```diff
@@ -145,6 +145,77 @@ export default {
       // Since this should only be called in a capture event (bottom up), we shouldn't need to stop propagation
       return getZIndex(this.$refs.content) >= this.getMaxZIndex()
     },
+    genActivator () {
+      if (!this.$slots.activator) return null
+      if (this.disabled) return this.$slots.activator
```

Comment:
This will cause different layout behaviour depending on if the dialog is disabled or not.